pos_tare_generate_barcode/static/src/xml/pos_tare_generate_barcode.xml

```diff
@@ -0,0 +1,47 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<templates id="template" xml:space="preserve">
+    <t t-name="TareScreenWidget">
+        <div class='tare-screen screen'>
+            <div class='screen-content'>
+                <div class='top-content'>
+                     <span class='button back'>
+                        <i class='fa fa-angle-double-left'></i>
+                        Back
+                    </span>
+                    <h1>Print a tare label</h1>
+                </div>
+                <div class="centered-content">
+                    <div class="pos-tare-label-container"></div>
+                    <div class="pos-directions-for-user">
+                        Set the pot on the scale and check the weight above
+                    </div>
+                    <div class='print-label'>
+                        Print
+                        <i class='fa fa-angle-double-right'></i>
+                    </div>
+                </div>
+            </div>
+        </div>
+    </t>
+
+    <t t-name="PosTareLabel">
+        <div class="pos-center-align">
+            <div class="pos-tare-paper">
+                <div class="pos-tare-label">
+                    <img t-att-src="'/report/barcode/EAN13/' + widget.get_barcode_data()" />
+                    <span class="caption">tare = <t t-esc="widget.get_tare_weight()" />kg</span>
```

Comment:
kg is hard coded.

Potential issue would be that the tare is on g and not kg. Any way to have uom if the tare configured somewhere ?

second point here is that it limits the use of this module on the kg metric system.

Reply:
I see that my proposition could to some uom conversion between the tare of the container and the uom of the contained product (not sure of that).

I agree to go ahead without change if you add a section explaining the limitation in the issue/limitation of the readme

Reply:
Good point, I'll fix, thank you.

Reply:
ok no issue with conversion, I just read the js code... and saw that you use the uom returned by the scale which is gr for our metric system. only limitation here is it doesn't support UK/US metric system. I'm fine with it !